config/config-builder/src/validator_config.rs

```diff
@@ -154,22 +154,26 @@ impl ValidatorConfig {
                 .and_then(|config| config.publishing_option.clone()),
         );
 
-        let waypoint = if self.build_waypoint {
+        let (waypoint, maybe_waypoint) = if self.build_waypoint {
```

Comment:
nit: maybe "maybe_waypoint" -> "test_waypoint"?

Reply:
well in all fairness all of these are test values, maybe_waypoint implies that it might or might not be there, it is commonly used in other parts of our code base.